src/delegate/types/CreateDelegateOfferModel.ts

```diff
@@ -1,6 +1,6 @@
 export interface User{
     Id: string,
-    Name: string
+    Email: string
```

Comment:
Door dit te veranderen komt dit niet overeen met de backend model waardoor email altijd null is.